tests/fixtures/saas/stripe_fixtures.py

```diff
@@ -153,7 +153,6 @@ def stripe_create_erasure_data(
                 "state": "CA",
             },
             "name": "Ethyca RTF",
-            "phone": "+19515551234",
```

Comment:
This test number is used for the access customer so I'm removing it from the auto-generated erasure customer